resume.md

```diff
@@ -1,49 +1,47 @@
+**Galileo Academy** | *June 2020*
+**University of College** | B.S. Degree ... *May 2024*
+* **GPA**:4.0 | 
+* **Coursework**: Introduction to Programming Data Science Foundations, Linear Algebra.
 
 ## SKILLS
-* **Languages**: Python, Java, C, SQL, ...
-* **Libraries/Frameworks**: Pandas, TensorFlow, PyTorch, ...
-* **Version Control**: Git, Bitbucket, Sourcetree, GithubDesktop
+* **Languages**: Python
+* **Design**: Figma, Adobe Illustrator, Procreate, Photoshop, Final Cut Pro
 * *...*
 
 ## EXPERIENCE
-**Software Engineer**: Company | City, State ... *Month 20XX - Present*
-* Implemented XYZ Microservice using TECHNOLOGY to delivery PRODUCT to 123 Customers Globally
-* Architected CORE INFRASTRUCTURE LIBRARY used across X teams to BLAH BLAH
-* I did X for Y to accomplish Z
-* ...
-
-**Software Engineer Intern**: Company | City, State ... *Month 20XX - Month 20XX*
-* Conducted A/B Test using FRAMEWORK to test EXPERIENCE in order to increase METRIC resulting in NUMBER. 
-* Utilized DEBUG TECHNOLOGY to resolve recurring crashes (X% of sessions) in beta before hitting production..
-* ...
-
-## PROJECTS
-**Project Name**: *Brief One Liner TL;DR* | tinyurl ... *Month 20XX - Month 20XX*
-* Developed PROJECT NAME in order to FEATURE solving PROBLEM for POPULATION
-* Launched PROJECT NAME to the Apple App Store resulting in X Downloads and Y Monthly Users
-* Beta Tested to X Number of users and collected Crash Data using analytics LIBRARY
-* ...
-
-**Project Name**: *Brief One Liner TL;DR* | tinyurl ... *Month 20XX - Month 20XX*
-* Contributed to OPEN SOURCE LIBRARY, a popular module for FEATURE, by closing X tasks and merging Y Pull Requests.
-* Created Demo Tutorial series to demonstrate FEATURE for early adopters prior to production push.
-* Managed App Version release cycles to maintain proper software versioning for major and minor app increments.
-* ...
+**Graphic Designer**: Innovative Design | Berkeley CA *Aug 2020 - Present*
+*	Created stylescapes in a branding project
+*	designed product mockups for small businesses
 
-## AWARDS/ACKNOWLEDGEMENTS
-* **COLLEGE Alumni Scholarship Recipient** - Alumni Scholarship Committee... *Month 20XX*
-* **1st Place Best Hack** - Hackathon Event... *Month 20XX*
+**Design Coordinator**: Associated Students of the University of California | Berkeley CA *August 2020 - Present*
+*	Created the ASUC brand book for design
+*	In the process of designing ASUC merch
+
+**Co-Founder**: CreAItivity | Bay Area CA *Jul 2019 - Present*
+* Youth-led non-profit seeking to educate underrepresented youth about AI
+*	Hosted a COVID-19 Challenge, 5 AI related workshops, and published 2 issues of tech-zines
+* (https://www.creaitivity.org/)
+
+**Public Health Researcher**: ING (startup) | San Francisco CA *August 2020 - Present*
+*	Researched and evaluated 800 Bay Area restaurants with ING’s healthy criteria
+*	Constructed the data library of evaluated restaurants and updated the ING criteria list
```

Comment:
How did you construct the library?  Is this excel?  is this using some Database?